src/parser/monk/mistweaver/modules/spells/InvokeChiJi.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+
+const debug = false;
+
+class InvokeChiJi extends Analyzer {
+  static dependencies = {
+    conbatants: Combatants
```

Comment:
typo

Reply:
Actually, I think its unused like in Yulon now that I look at it